packages/event-handler/src/rest/converters.ts

```diff
@@ -45,4 +49,45 @@ export const proxyEventToWebRequest = (event: APIGatewayProxyEvent) => {
     headers,
     body: createBody(event.body, event.isBase64Encoded),
   });
-}
+};
+
+export const responseToProxyResult = async (
+  response: Response
+): Promise<APIGatewayProxyResult> => {
+  const headers: Record<string, string> = {};
+  const multiValueHeaders: Record<string, Array<string>> = {};
+
+  for (const [key, value] of response.headers.entries()) {
+    const values = value.split(',').map((v) => v.trimStart());
+    if (values.length > 1) {
+      multiValueHeaders[key] = values;
+    } else {
+      headers[key] = value;
+    }
+  }
+
+  return {
+    statusCode: response.status,
+    headers,
+    multiValueHeaders,
+    body: await response.text(),
+    isBase64Encoded: false,
```

Comment:
I've hardcoded this for now but we will need to figure out how to deal with this